package.json

```diff
@@ -1,51 +1,64 @@
 {
```

Comment:
@pavandv what about `"dev": "rm -rf lib && concurrently \"yarn sync-files --watch\" \"yarn compile --watch\" \"yarn demo\"",` script?

How to run repo in dev mode?

Reply:
Figured a way to resolve the issue in another way.. Working on it.

Reply:
To **serve applications**, one has to run `npm/yarn start`.

To **build packages**, one has to run `npm run build` or `yarn build`